src/Acts/CamdramBundle/Entity/Person.php

```diff
@@ -530,8 +530,8 @@ public function getIndexDate()
     {
         $latest = null;
         foreach ($this->getRoles() as $role) {
-            if ($role->getShow() && (!$latest || $role->getShow()->getStartAt() > $latest) && $role->getShow()->getStartAt()) {
-                $latest = $role->getShow()->getStartAt();
+            if ($role->getShow() && (!$latest || $role->getShow()->getEndAt() > $latest) && $role->getShow()->getEndAt()) {
```

Comment:
I didn't change this because I'm not familiar enough with PHP to be sure in my thinking, but wouldn't `if ($role->getShow() && $role->getShow()->getEndAt() && (!$latest || $role->getShow()->getEndAt() > $latest))` be more correct? I'm assuming `$role->getShow()->getEndAt()` is a not-null check?